coral/src/app/pages/configuration/clusters/index.tsx

```diff
@@ -1,12 +1,32 @@
 import { PageHeader } from "@aivenio/aquarium";
+import add from "@aivenio/aquarium/icons/add";
+import { useNavigate } from "react-router-dom";
 import PreviewBanner from "src/app/components/PreviewBanner";
+import { useAuthContext } from "src/app/context-provider/AuthProvider";
 import Clusters from "src/app/features/configuration/clusters/Clusters";
+import { Routes } from "src/app/router_utils";
 
 const ClustersPage = () => {
+  const navigate = useNavigate();
+  const {
+    permissions: { addDeleteEditClusters },
```

Comment:
Small thought -> in this case, I would maybe not destructure, but use `permissions.addNewCluster` in the condition, because than it's very understandable already at line 21. But that's super nitpicky and not needed!

Reply:
Makes sense to me ^^